test/e2e/traceflow_test.go

```diff
@@ -36,10 +36,9 @@ type testcase struct {
 	expectedResults []v1alpha1.NodeResult
 }
 
-// TestTraceflow verifies if traceflow can trace intra/inter nodes traffic with some NetworkPolicies set.
-func TestTraceflow(t *testing.T) {
-	skipIfProviderIs(t, "kind", "Inter nodes test needs Geneve tunnel")
-	skipIfNumNodesLessThan(t, 2)
+// TestTraceflow_IntraNode verifies if traceflow can trace intra node traffic with some NetworkPolicies set.
+func TestTraceflow_IntraNode(t *testing.T) {
+	skipIfNumNodesLessThan(t, 1)
```

Comment:
I would say remove this? `numNodes` include the control-plane Node, so less than 1 is not possible...

Reply:
fixed